GTG/core/info.py.in

```diff
@@ -36,7 +36,7 @@
 TRANSLATE_URL = "https://github.com/getting-things-gnome/gtg/"
 REPORT_BUG_URL = "https://github.com/getting-things-gnome/gtg/issues/new"
 EMAIL = "gtg-contributors@lists.launchpad.net"
-VERSION = '0.4.0'
+VERSION = '@VCS_TAG@'
```

Comment:
When a release gets made, it'll be `VERSION = 'v0.5'`, which diverges from the "expected" format.
Maybe save it in a new variable and use some python code here to determine what to use?
This could also be fine, but I'd rather point it out.

Reply:
I'm not sure what you mean with that. Does the version format change in the next version?

Reply:
Well, `git describe` uses (annotated) tags as a base, and when on a annotated tag, it directly uses the tag name. However, the format of those are `vX.Y`, see: https://github.com/getting-things-gnome/gtg/tags , which as you can see differs from the format uses here (`X.Y.Z`). If the maintainers/others are OK with that, tehn I think it could stay as-is. I haven't checked how it looks like on the GUI.

Reply:
It looks like this now. Do you know where it gets the branch name from? I couldn't find any other use of git in the source.

![Screenshot from 2020-11-19 21-13-14](https://user-images.githubusercontent.com/10071947/99721589-c05e9700-2aaf-11eb-9b86-3a923941b93c.png)

Reply:
It is the title and the "branch" is actually the folder name here: https://github.com/getting-things-gnome/gtg/blob/d7bbcd5b2a9d7d54b2b28278e5bbf6fd264b302b/scripts/debug.sh#L48 and https://github.com/getting-things-gnome/gtg/blob/master/GTG/gtg.in#L102

I also looked a bit and it seems `VERSION` is only used twice, for displaying not parsing, so this would be fine I think. I'll however want others feedback, whenever it would be OK to change the "format". I guess I might also be too picky here.